_docs-sources/intro/environment-setup/recommended_tools.md

```diff
@@ -0,0 +1,84 @@
+# Recommended Tools
+
+* [AWS CLI](https://aws.amazon.com/cli/) - The AWS CLI is a powerful tool that lets perform most of the actions that you can do via the AWS Console.
```

Comment:
Let's use em dashes instead of hyphens for all of these.
```suggestion
* [AWS CLI](https://aws.amazon.com/cli/) — The AWS CLI is a powerful tool that lets perform most of the actions that you can do via the AWS Console.
```

Reply:
You can copy paste, but you can also type this character with `OPTION + hyphen`

Reply:
Addressed in 10d5df4